lib/alerts/supervisor.ex

```diff
@@ -17,8 +17,7 @@ defmodule Alerts.Supervisor do
 
         _ ->
           [
-            Alerts.CacheSupervisor,
-            Alerts.BusStopChangeSupervisor
```

Comment:
blocking: The logic in this statement can be simplified now.

I'm also curious why move this supervisor from here to `application.ex`? Though, now that both these supervisors are down to one child each, maybe we can do away with both and just put the children in `application.ex`.....